### ISSUE TITLE
Extra alignment issue

### ISSUE THREAD

**Issue by AndBerna** [jeslago/optidef#23]
Hi, 

when i am using the extra alignment feature the equation (number) jumps to the next line even though there is still enough space on the line.

I am using the constraint style 2 and the nocomma paramenter when importing the package.

Thanks for all

**Comment by AndBerna** [comment]
Example:

\addConstraint{q^{hp}_{t,n}}{=COP_n   p^{hp}_{t,n}, \label{eq:base_HP_n_COP}}{\forall \ t \ \forall \ n}

**Comment by AndBerna** [comment]
Any update?

Thanks in advance

**Comment by jeslago** [comment]
Hi,

can you provide a full example? Does it only occur when using the extra alignment feature?

The numbering of the equations is managed by a third library (the subequations). In principle, this library moves the numbering down whenever the space between equation and number is small. 

Can you provide an example so I can help you better?

**Comment by AndBerna** [comment]
Hi,

I made a quick solution it by adding a \vspace{} meanwhile. 

I was working with the "long" format (mimize, subject to) but I had to change it to the short version. And I have just checked that the  problem does not happen when I when working in such format.

I tried to change the format to the long style and now it doesnt have the problem there  either . "Unfortunately" I am not able to reproduce it any more. =D 

Maybe there was a package that was crashing.

Sorry for taking part of  your time (dont know what happened)


Thanks =) =)
